packages/amplify_auth_plugin_interface/lib/src/SignIn/AuthProvider.dart

```diff
@@ -13,15 +13,9 @@
  * permissions and limitations under the License.
  */
 
-
-
 /*
   !!IMPORTANT!! 
   Dart enums are 0-based indices; new values must be added to the end of the list.
   !!IMPORTANT!! 
 */
-enum AuthProvider {
-  google,
-  facebook,
-  login_with_amazon,
-} 
\ No newline at end of file
+enum AuthProvider { google, facebook, amazon, apple }
```

Comment:
FAANG order, perhaps!

```suggestion
enum AuthProvider { facebook, amazon, apple, google }
```